ci-operator/config/containers/libpod/master.yaml

```diff
@@ -23,6 +23,10 @@ tests:
   as: lint
   commands: ARTIFACT_DIR=/tmp/artifacts make lint
   from: test-bin
+- artifact_dir: /tmp/artifacts
+  as: rpm verification 
```

Comment:
Eek -- this needs to be a single word. @droslean let's validate that

Reply:
fixed